monkey/monkey_island/cc/services/infection_lifecycle.py

```diff
@@ -25,11 +25,16 @@ def set_stop_all(time: float):
 
 def should_agent_die(guid: int) -> bool:
     monkey = Monkey.objects(guid=str(guid)).first()
-    return _is_monkey_marked_dead(monkey) or _is_monkey_killed_manually(monkey)
+    return _should_agent_stop(monkey) or _is_monkey_killed_manually(monkey)
```

Comment:
I think this can just be removed.

Reply:
I'll do some further investigation, but `_is_monkey_killed_manually` is used and `_should_agent_stop` could be used to kill single monkeys?